md/customize-living-application-layout.md

```diff
@@ -5,18 +5,14 @@
 :::
 
 ::: warning
-Since Bonita BPM 7.3, we have made some changes in the default layout. You will have to remove `layout.css` from the page assets to make the following instructions relevant.
+Since 7.5 we made some improvements on the default layout.
+We recommand that you re-create your custom layout based on Bonita 7.5.0+ default layout.
```

Comment:
Since the previous version we made some improvements on the default layout.
We recommand that you re-create your custom layout based on Bonita 7.6.0+ default layout.